src/linkmod.rs

```diff
@@ -33,11 +33,9 @@ struct Mod {
 /// Creates an embed based on the recieved mod data. {{{1
 /// Returns true if successful.
 fn make_mod_embed(modification: Mod, message: &Message) -> bool {
-    let mut tag_str = String::from("Not tagged.");
-    if let Some(ref tag) = modification.tag {
-        tag_str = tag.clone();
-    }
-    let result = message.channel_id.send_message(move |a| {
+    let tag_str = modification.tag.as_ref().map(String::as_str).unwrap_or("Not tagged.");
```

Comment:
Instead of allocating space for a new string on the heap, one can instead just use a &str ref.